src/main/resources/schema/moderne-organizations.graphqls

```diff
@@ -27,8 +27,54 @@ type Organization {
     commitOptions: [CommitOption!]!
 
     parent: Organization
+
+    """
+    The dashboard for this organization.
+    """
+    dashboard: DashboardConfiguration
+}
+type DashboardConfiguration{
+    """
+    Available/recommended upgrades and migrations.
+    """
+    upgradesAndMigrations: [RecipeRunRequest!]
+
+    """
+    Recommended visualizations to run on this organization.
+    """
+    visualizations: [VisualizationRequest!]
+
+    """
+    Security recipes to run for this organization.
+    """
+    security: [RecipeRunRequest!]
 }
 
+type VisualizationRequest{
+    recipeId: String!
+    recipeOptions: [Option!]!
+    visualizationId: String!
+    visualizationOptions: [Option!]!
+}
+
+type RecipeRunRequest{
+    recipeId: String!
+    options: [Option!]!
+}
```

Comment:
I don't think `Request` is exactly right because this is a "pull" model. In a previous PR, i had called these `DashboardVisualization` and `DashboardRecipeRun`, which, while also not exactly right, better indicates that these are not requests sent to a service.

I do understand the rationale -- this is the central team / org lead requesting that we run these recipes on their organization. So I think I could be convinced about this.

Reply:
I think you are right. The customers do not care that this is a "request" they are configuring their dashboard.